src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,18 @@
+import { Person } from '../../types/Person';
+import { DropdownItem } from '../DropdownItem';
+
+interface Props {
+  people: Person[];
+}
+
+export const DropDownMenu: React.FC<Props> = ({ people }) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map((person, index) => (
+          <DropdownItem person={person} key={index + 1} />
```

Comment:
You're using `index + 1` as a key for the `DropdownItem` component. This is not recommended, as keys should be stable, predictable, and unique for optimal performance. If the order of items changes, this might lead to unexpected behaviour. It's better to use unique and stable identifiers from the data itself, like `person.id` or similar. Here is a [useful article](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052) that explains why generating keys on render can cause issues.